src/App.js

```diff
@@ -2,18 +2,30 @@ import { React, useEffect, useState } from "react";
 import ToDoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
-const useSemiPersistentState = () => {
+function App() {
   const localData = JSON.parse(localStorage.getItem('savedTodoList'))
-  const [todoList, setTodoList] = useState((localData.length ? localData : []))
+  const [todoList, setTodoList] = useState([])
```

Comment:
There is still a bug here. You can reference my comment from before on how to fix this at the link below. You need to handle the potential of localData being undefined when it is declared. Then you can just use it everywhere without performing checks deeper in your code.

https://github.com/travisdjones78/React/pull/10#discussion_r1155990884